chemprop/cli/train.py

```diff
@@ -650,6 +641,63 @@ def build_splits(args, format_kwargs, featurization_kwargs):
     return train_data, val_data, test_data
 
 
+def summarize(args, dataset: _MolGraphDatasetMixin) -> tuple[list, list]:
+    if args.task_type == "spectral":
+        pass
+    elif args.task_type in ["regression", "regression-mve", "regression-evidential"]:
+        y = dataset.Y
+        y_mean = y.mean()
+        y_std = y.std()
+        y_median = np.median(y)
+        mean_dev_abs = np.abs(y - y_mean)
+        frac_1_sigma = (mean_dev_abs < y_std).sum() / len(y)
+        frac_2_sigma = (mean_dev_abs < 2 * y_std).sum() / len(y)
+        column_headers = ["Statistic", "Value"]
+        table_rows = [
+            ["Num. datapoints", f"{len(y)}"],
+            ["Mean", f"{y_mean:0.2f}"],
+            ["Std. dev.", f"{y_std:0.2f}"],
+            ["Median", f"{y_median:0.2f}"],
+            ["% within 1 s.d.", f"{frac_1_sigma:0.0%}"],
+            ["% within 2 s.d.", f"{frac_2_sigma:0.0%}"],
+        ]
+        return (column_headers, table_rows)
```

Comment:
does this work `nan`s in the dataset? Just checking.

Reply:
```
>>> import numpy as np
>>> a = np.array([1, 2, np.nan])
>>> a.mean()
nan
>>> np.nanmean(a)
1.5
```

Reply:
I fixed the nan issue